bundles/org.openhab.binding.darksky/src/main/java/org/openhab/binding/darksky/internal/handler/DarkSkyWeatherAndForecastHandler.java

```diff
@@ -136,9 +136,10 @@ public void initialize() {
         try {
             location = new PointType(config.location);
         } catch (IllegalArgumentException e) {
-            location = null;
+            logger.warn("Error parsing 'location' parameter: {}", e.getMessage());
```

Comment:
I usually request such logs to be removed, as thing status changes are logged anyway. Why can't we add the message to the thing status?

Reply:
Yes, that is right. I thought about that too. But why I did not do it was because the error message of the exception is not localized. It is English. The `ThingStatus` descriptions in this binding are translatable which could look a little bit weird if you have a mixture of languages in it.

I am brainstorming about a solution for localized exception messages in OHC for a while now. Do not know if it is really worth the effort to spend time on it.

Reply:
I just had a look at the `PointType` class. As far as I can see there are three cases where an IAE is thrown: empty string, less than two parts, more than three parts, so all are related to the format. What additional information do we get by logging the message? Probably the thing status message shpuld be a bit more clear that the format of `location` is wrong.

Reply:
How much effort should I spend on a [dead](https://community.openhab.org/t/dark-sky-yawb4oh2-yet-another-weather-binding-for-openhab-2/69968/83?u=cweitkamp) binding?